pom.xml

```diff
@@ -705,8 +707,18 @@
             <dependency>
                 <groupId>org.jboss.aesh</groupId>
```

Comment:
If my understanding is right, we no longer need this `org.jboss.aesh` dependency now.

Reply:
@mingjliu9 It's unfortunately still needed as keycloak-admin-cli and keycloak-client-registration-cli are still using it. Both CLIs are packed in server distribution in directory KEYCLOAK_HOME/bin/client/ in the "fat" jars, which among other things, contain the classes of old aesh. I've created https://issues.jboss.org/browse/KEYCLOAK-7737 . 

Fortunately org.jboss.aesh is not needed in jboss modules, so IMO KEYCLOAK-7737 is not a blocker for this PR, but can be done later.